bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/tWZj_NLO_5f/tWZj_NLO_5f_run_card.dat

```diff
@@ -0,0 +1,285 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update full_run_card                                          *
+#*********************************************************************
+#
+#*******************
+# Running parameters
+#*******************
+#
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000	= nevents ! Number of unweighted events requested
+  0	= iseed ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+  1	= lpp1 ! beam 1 type
+  1	= lpp2 ! beam 2 type
+  6800.0	= ebeam1 ! beam 1 total energy in GeV
+  6800.0	= ebeam2 ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf	= pdlabel ! PDF set
+  325300	= lhaid ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
Because the b quark is not in the multiparticle definition, the default will be 4FS in this case. Given the process we are trying to generate is always generated with a b quark in the initial state, a 4FS PDF will give a cross-section of 0. Explicitly defining the PDF like this was a work around for this problem.

Reply:
ok, i see. it makes sense.